src/ray/core_worker/task_manager.h

```diff
@@ -47,6 +48,8 @@ class TaskFinisherInterface {
       const std::vector<ObjectID> &inlined_dependency_ids,
       const std::vector<ObjectID> &contained_ids) = 0;
 
+  virtual void MarkDependenciesResolved(const TaskID &task_id) = 0;
```

Comment:
Personal preference, but why don't we use `OnDependenciesResolved`? Mark sounds like we will have some operations for the task id